src/service/persister.ts

```diff
@@ -162,13 +168,59 @@ export class Persister<T extends Persistable> implements ILoadAll<T> {
     public persist = (report: T) => {
         this.collection.then(coll => {
             this._saver(report);
+            if (["fv","md","msg","mt","pos","qp-sub","tbp","osr","tsv"].indexOf(this._dbName)>-1)
+              coll.deleteMany({ time: { $exists:true } }, err => {
+                  if (err)
+                      this._log.error(err, "Unable to deleteMany", this._dbName, report);
+              });
```

Comment:
truncate collections to 1 single record to save space in very very long runs; only trades table persist unlimited records